src/components/aboutUs/components/TeamProfile.js

```diff
@@ -4,37 +4,36 @@ import { Avatar, Card, Icon, List } from 'antd';
 import { ABOUT_US, SOCIAL_MEDIA, TEAM_INFO } from '../constants';
 
 const { Meta } = Card;
-const { 
-    CLASSNAMES: { AVATAR, TEAM_CARD },
-    STRINGS: { SQUARE }, 
+const {
+    CLASSNAMES: { AVATAR, TEAM_CARD, TEAM_HEADER },
+    STRINGS: { SQUARE },
+    TEXTS: { HEADER_TEXT },
 } = ABOUT_US;
 
 const actionList = SOCIAL_MEDIA.map(social => (<Icon type={social} key={social} />));
 
 class TeamProfile extends React.Component {
     render() {
         return(
-            <List  
-                bordered={false}
+            <List
                 dataSource={TEAM_INFO}
-                grid={{ gutter: 12, lg: 4, md: 1.7, sm: 1.8, xs: 1.7, xxl: 3 }}
```

Comment:
Do not use decimal places when dealing with a grid. Only whole numbers.

Reply:
why is decimal not used for gird?

Reply:
yea

Reply:
`...lg: 4, md: 1.7, sm: 1.8, xs: 1.7, xxl: 3 }`
so what is this piece of code saying? from left to right, it is saying for large screens show 4 items, for middle sized screens show 1.7 items, ...for extra extra large screen show 3 items. You can't have 1.7 items, so what does the browser do, it adds extra white space to met the request that you made of it. Which leads to messed up UI